tdms/tests/system/test_fdtd.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+
+import pytest
+from utils import HDF5File, download_data, run_tdms, work_in_zipped_dir
+
+ZIP_PATH = Path(
+    os.path.dirname(os.path.abspath(__file__)), "data", "arc_example_fdtd.zip"
+)
+
+if not ZIP_PATH.exists():
+    download_data(
+        "https://zenodo.org/record/7148198/files/arc_example_fdtd.zip", to=ZIP_PATH
+    )
+
+
+@pytest.mark.fdtd_build_only
+@work_in_zipped_dir(ZIP_PATH)
+def test_fs():
+    """
+    Ensure that the free space FDTD output matches the reference. Checks
+    that the output .mat file (with a HDF5 format) contains tensors with
+    relative mean square values within numerical precision of the reference.
+
+    Note: At the moment (due to MATLAB) this test only works if tdms is compiled with
+    """
+
+    # misleading filename here
```

Comment:
Agreed - can we (or get Peter to if he's the only one with access) get these named a bit more consistently? (`fdtd` over `pstd` of course)

Reply:
It's me.
We can, or we wait for peter to give us a meaningful test before re-uploading.

Reply:
Let's leave it as it is for now then, and when we get a more meaningful test update the names at the same time as we re-upload

Reply:
Yeah there is also
* #116